integrationExamples/gpt/hello_world.html

```diff
@@ -22,11 +22,29 @@
 
         // Replace this object to test a new Adapter!
         bids: [{
-          bidder: 'appnexus',
+          bidder: 'trustx',
```

Comment:
This page is just for testing, changes to this file aren't required in PRs so the changes here can be dropped